build.ps1

```diff
@@ -11,6 +11,8 @@ param (
     [string]$NuGetPFXPath,
     [switch]$SkipXProj,
     [switch]$SkipCSProj,
```

Comment:
I'm worried about amount of switches especially those that overlap. Build script alone won't give you all possible permutations of build tasks. Its purpose is to provide end-to-end build and test experience for the most common scenarios, including CI and local dev. For that purpose it should be simple and straightforward.

If you want to invoke a single task you can dot-source common.ps1 and call a function you need directly from PS console. In your example it would be like this:
```PS
. .\build\common.ps1
Test-ClientsProjetcs
```

Given all that can we assume $SkipCSProj = $SkipDev14 + $SkipDev15 and remove it?